docs/src/pages/components/data-grid/selection/selection.md

```diff
@@ -28,13 +28,14 @@ Row selection can be performed with a simple mouse click, or using the [keyboard
 ### Single row selection
 
 Single row selection is enable by default with the `DataGrid` component.
+To unselect a row, hold the <kbd class="key">CTRL</kbd> or <kbd class="key">Shift</kbd> (use <kbd class="key">⌘ Command</kbd> on macOS) key and click on it.
```

Comment:
I would propose we only mention CTRL per https://material-ui.com/components/data-grid/accessibility/#key-assignment-conventions
```suggestion
To unselect a row, hold the <kbd class="key">CTRL</kbd> or <kbd class="key">Shift</kbd> key and click on it.
```

It's more or less based on https://www.w3.org/TR/wai-aria-practices/#kbd_common_conventions.

Reply:
I would vote against using <kbd>Shift</kbd>. You would end up unselecting the current row if you are in multi-select mode and slightly miss your click target. At least, macOS doesn't support it not 

```suggestion
To unselect a row, hold the <kbd class="key">CTRL</kbd> (use <kbd class="key">⌘ Command</kbd> on macOS) key and click on it.
```

Reply:
Yes, <kbd>Shift</kbd> is generally used for extending the selection. It doesn't makes sense but `isMultipleKeyPressed` supports it. I'll open a follow-up PR to fix the helper.

Reply:
`isMultipleKeyPressed` is only used for the multi-header sorting. Which is an interesting problem. on AG Grid only <kbd>Shift</kbd> is handled. But on handsontable, a click on the header selects the whole column, so they only implement the multi-sorting wth <kbd>Ctrl</kbd>. IHMO the isMultipleKeyPressed abstraction is wrong, it should be handled on case by case.